.github/workflows/scripts/kinds/verify-kinds.go

```diff
@@ -1,116 +1,57 @@
 	"context"
 	"errors"
 	"fmt"
-	"io"
-	"net/http"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
 	"os"
 	"path/filepath"
 	"regexp"
-	"strconv"
 	"strings"
-	"testing/fstest"
 
 	"cuelang.org/go/cue"
 	cueformat "cuelang.org/go/cue/format"
-	"github.com/google/go-github/github"
 	"github.com/grafana/codejen"
-	"github.com/grafana/grafana/pkg/codegen"
-	"github.com/grafana/grafana/pkg/cuectx"
-	"github.com/grafana/grafana/pkg/plugins/pfs"
-	"github.com/grafana/grafana/pkg/plugins/pfs/corelist"
-	"github.com/grafana/grafana/pkg/registry/corekind"
-	"github.com/grafana/kindsys"
-	"github.com/grafana/thema"
-	"golang.org/x/oauth2"
+	"github.com/grafana/grafana/pkg/registry/schemas"
 )
 
-const (
-	GITHUB_OWNER = "grafana"
-	GITHUB_REPO  = "kind-registry"
-)
+var nonAlphaNumRegex = regexp.MustCompile("[^a-zA-Z0-9 ]+")
 
 // main This script verifies that stable kinds are not updated once published (new schemas
 // can be added but existing ones cannot be updated).
-// If the env variable CODEGEN_VERIFY is not present, this also generates kind files into a
-// local "next" folder, ready to be published in the kind-registry repo.
+// It generates kind files into a local "next" folder, ready to be published in the kind-registry repo.
 // If kind names are given as parameters, the script will make the above actions only for the
 // given kinds.
 func main() {
-	var corek []kindsys.Kind
-	var compok []kindsys.Composable
-
-	kindRegistry, err := NewKindRegistry()
-	defer kindRegistry.cleanUp()
-	if err != nil {
-		die(err)
-	}
-
-	// Search for the latest version directory present in the kind-registry repo
-	latestRegistryDir, err := kindRegistry.findLatestDir()
-	if err != nil {
-		die(fmt.Errorf("failed to get latest directory for published kinds: %s", err))
-	}
-
-	errs := make([]error, 0)
-
-	// Kind verification
-	for _, kind := range corekind.NewBase(nil).All() {
-		name := kind.Props().Common().MachineName
-		err := verifyKind(kindRegistry, kind, name, "core", latestRegistryDir)
-		if err != nil {
-			errs = append(errs, err)
-			continue
-		}
-
-		corek = append(corek, kind)
-	}
+	// File generation
+	jfs := codejen.NewFS()
+	outputPath := filepath.Join(".github", "workflows", "scripts", "kinds")
 
-	for _, pp := range corelist.New(nil) {
-		for _, kind := range pp.ComposableKinds {
-			si, err := kindsys.FindSchemaInterface(kind.Def().Properties.SchemaInterface)
-			if err != nil {
-				errs = append(errs, err)
-				continue
-			}
-
-			name := strings.ToLower(fmt.Sprintf("%s/%s", strings.TrimSuffix(kind.Lineage().Name(), si.Name()), si.Name()))
-			err = verifyKind(kindRegistry, kind, name, "composable", latestRegistryDir)
-			if err != nil {
-				errs = append(errs, err)
-				continue
-			}
-
-			compok = append(compok, kind)
-		}
-	}
+	corekinds, err := schemas.GetCoreKinds()
+	die(err)
 
-	die(errs...)
+	composableKinds, err := schemas.GetComposableKinds()
+	die(err)
 
 	if _, set := os.LookupEnv("CODEGEN_VERIFY"); set {
```

Comment:
Is this still useful? It looks like there is no longer any "verification" done by this script.

Reply:
It only avoids to generate the next folder, but only that.

Reply:
I just deleted since it doesn't "verify" anything.